homeassistant/components/onvif/translations/ca.json

```diff
@@ -31,11 +24,13 @@
                 },
                 "title": "Selecci\u00f3 de dispositiu ONVIF"
             },
-            "manual_input": {
+            "configure": {
```

Comment:
Please don't change translations of foreign languages, these are managed by our translation platform.

Revert all changes to all foreign language files.